PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -68,19 +68,13 @@ def __eq__(self, other: object) -> bool:
         :rtype: bool
         :return: True if seeds are the same, False otherwise.
         """
-        # > hmac.compare_digest` uses an approach designed to prevent timing
-        # > analysis by avoiding content-based short circuiting behaviour, making
-        #  > it appropriate for cryptography
-        # > https://docs.python.org/3.7/library/hmac.html#hmac.compare_digest
-        #
-        # > Note: If a and b are of different lengths, or if an error occurs,
-        # > a timing attack could theoretically reveal information about the types
-        #  > and lengths of a and b—but not their values.
-        #
-        # Type and length of seeds is known to the attacker, but not the value of expected seed.
-        if not isinstance(other, Seed):
+        if not isinstance(other, Seed) or len(self) != len(other):
             return False
```

Comment:
This approach in general would leak information about length of a secret. But in our case, the attacker knows correct length of seed, so it's fine. :)

Reply:
I've replaced quick return, please, check one more time.

Reply:
If it's valid instance of `Seed` then the length is fixed `SEED_LEN`.

Reply:
Removed extra length check, thanks!